### PR TITLE
mrc-1434 Translations for custom colour scales

### PR DESCRIPTION
**Pull request by EmmaLRussell**

### CODE REVIEW

File: src/app/static/src/tests/components/plots/mapLegend.test.ts [hunk 1]
```diff
@@ -25,7 +34,7 @@ describe("Map legend component", () => {
                 customMax: 2.5
             },
             colourRange
-        }
+        }, localVue
```
Comment (hillalex) [inline src/app/static/src/tests/components/plots/mapLegend.test.ts:37]: why add this localVue here? and actually this wrapper is only used in 1 test, so probably makes sense to declare it within the scope of that test to keep things clear

Comment (hillalex) [reply src/app/static/src/tests/components/plots/mapLegend.test.ts:37]: oops, merged this (code and functionality all looks great) but meant to raise this comment about the test code @EmmaLRussell

Comment (EmmaLRussell) [reply src/app/static/src/tests/components/plots/mapLegend.test.ts:37]: I think that might have been a relic from an earlier version of the test and isn't actually needed, will remove in separate branch. .

Comment (hillalex) [reply src/app/static/src/tests/components/plots/mapLegend.test.ts:37]: nice, thanks

**Pull request merged**